src/server/grpc/p2pool.rs

```diff
@@ -40,22 +40,18 @@ use crate::{
         p2p,
         stats_store::StatsStore,
     },
-    sharechain::{block::Block, BlockValidationParams, ShareChain, SHARE_COUNT},
+    sharechain::{block::Block, error::BlockConvertError, BlockValidationParams, ShareChain, SHARE_COUNT},
 };
 
-const LOG_TARGET: &str = "p2pool::server::grpc::p2pool";
+const LOG_TARGET: &str = "tari::p2pool::server::grpc::p2pool";
 
 pub fn min_difficulty(consensus_manager: &ConsensusManager, pow: PowAlgorithm, height: u64) -> Difficulty {
     let consensus_constants = consensus_manager.consensus_constants(height);
     match pow {
-        PowAlgorithm::RandomX => consensus_constants
-            .min_pow_difficulty(pow)
-            .checked_div_u64(MIN_DIFFICULTY_REDUCTION_RATE)
-            .expect("checked_div_u64 should only fail on div by 0"),
-        PowAlgorithm::Sha3x => consensus_constants
-            .min_pow_difficulty(pow)
-            .checked_div_u64(MIN_DIFFICULTY_REDUCTION_RATE)
-            .expect("checked_div_u64 should only fail on div by 0"),
+        PowAlgorithm::RandomX => consensus_constants.min_pow_difficulty(pow),
+        PowAlgorithm::Sha3x => {
+            Difficulty::from_u64(consensus_constants.min_pow_difficulty(pow).as_u64() * 400).expect("Bad difficulty")
```

Comment:
I suspect we still need to divide here, or make new constants. The constants up top is created for full blocks 2 min blocks